### PR TITLE
Task #10788: Advanced login

### PR DESCRIPTION
**Pull request by huynhlv**
Screenshot from 2019-04-18 20-37-53
https://edu-redmine.sun-asterisk.vn/issues/10788

### CODE REVIEW

**Comment by huynhlv** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class User < ApplicationRecord
   has_many :bookings, dependent: :destroy
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+  attr_accessor :remember_token
```
Comment (VuThiTranVan) [inline app/models/user.rb]: Line code này khai báo lên trên các association nhé

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,4 @@
 <%= @user.lastname %>, <%= @user.email %>
+<% if logged_in? %>
+  <%= link_to "Log out", logout_path, method: :delete %>
```
Comment (VuThiTranVan) [inline app/views/users/show.html.erb]: I18n nhé

**Comment by huynhlv** [comment]
ready

**Pull request merged**